static/js/publisher-pages/pages/Releases/Releases.tsx

```diff
@@ -0,0 +1,66 @@
+import { useParams } from "react-router-dom";
+import { useQuery } from "react-query";
+import { Strip } from "@canonical/react-components";
+
+import SectionNav from "../../components/SectionNav";
+import Release from "./Release";
+
+function Releases(): JSX.Element {
+  const { snapId } = useParams();
+  const { isLoading, isFetched, data } = useQuery({
+    queryKey: ["releases"],
+    queryFn: async () => {
+      const response = await fetch(`/api/${snapId}/releases`);
+
+      if (!response.ok) {
+        throw new Error("There was a problem fetching releases data");
+      }
+
+      const responseData = await response.json();
+
+      if (!responseData.success) {
+        throw new Error("There was a problem fetching releases data");
+      }
+
+      return responseData.data;
+    },
+  });
+
+  return (
+    <>
+      <h1 className="p-heading--4">
+        <a href="/snaps">My snaps</a> / <a href={`/${snapId}`}>{snapId}</a> /
```

Comment:
nitpick: can you use Link component in the vanilla framework? https://canonical.github.io/react-components/?path=/docs/components-link--docs

Reply:
Done